kedro-datasets/RELEASE.md

```diff
@@ -1,5 +1,6 @@
 # Upcoming Release
 ## Major features and improvements
+* Removed support for Python 3.9
```

Comment:
Don't forget to add your name to the contributor list 🙂

Reply:
Will do:)